audioswitch/src/main/java/com/twilio/audioswitch/selection/AudioDeviceSelector.kt

```diff
@@ -218,7 +231,7 @@ class AudioDeviceSelector {
      *
      * @param audioDevice The [AudioDevice] to use
      */
-    fun selectDevice(audioDevice: AudioDevice?) {
+    fun selectDevice(audioDevice: AudioDevice) {
```

Comment:
Looks like the arg isn't nullable anymore. It should be based on the method documentation provided above.

Reply:
There could be a test for this.

`setting_the_selected_device_back_to_null_prefers_bluetooth` for example.